tests/integration/test_remoted/test_configuration/test_basic_configuration_syslog_allowed_ips_invalid.py

```diff
@@ -22,14 +22,22 @@
     {'ALLOWED': '127.0.0.0.0', 'DENIED': '192.168.1.1'},
     {'ALLOWED': 'Testing', 'DENIED': '192.168.1.1'},
     {'ALLOWED': '127.0.0.0/7890', 'DENIED': '192.168.1.1'},
-    {'ALLOWED': '127.0.0.0/7890', 'DENIED': '192.168.1.1.1'}
+    {'ALLOWED': '127.0.0.0/7890', 'DENIED': '192.168.1.1.1'},
+    {'ALLOWED': '::1::1', 'DENIED': 'fe80::1003:889f:a584:0101'},
+    {'ALLOWED': 'Testing', 'DENIED': 'fe80::1003:889f:a584:0101'},
+    {'ALLOWED': '::1/512', 'DENIED': 'fe80::1003:889f:a584:0101'},
+    {'ALLOWED': '::1/512', 'DENIED': 'fe80::1003:889f:a584::0101'}
```

Comment:
In this test, we only need to check invalid allowed-ips. Please remove invalid denied ips cases.